integrated_channels/sap_success_factors/transmitters/content_metadata.py

```diff
@@ -62,6 +62,10 @@ def transmit(self, payload, **kwargs):
                     items_to_update.pop(content_metadata_id, None)
                     items_to_delete.pop(content_metadata_id, None)
 
+                # SAP servers throttle incoming traffic, If a request fails than the subsequent would fail too,
+                # So, no need to keep trying and failing. We should stop here and retry later.
+                break
```

Comment:
glad that this is simple! however, we need to update items_to_create, items_to_update, and items_to_delete to reflect what items have actually been sent to the api, because below we commit them to the db. with this, the calls that get skipped after an error would be recorded as having been sent.